src/test/java/org/logstash/input/DeadLetterQueueInputPluginTests.java

```diff
@@ -47,37 +51,39 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void test() throws Exception {
-        DeadLetterQueueWriter queueWriter = new DeadLetterQueueWriter(dir, 100000000, 10000000);
-        DLQEntry entry = new DLQEntry(new Event(), "test", "test", "test");
-        for (int i = 0; i < 10000; i++) {
-            queueWriter.writeEntry(entry);
+    public void testIT() throws Exception {
```

Comment:
`test` wasn't a good name, maybe some more descriptive name, like 
```suggestion
    public void testHappyPath() throws Exception {
```

or `testNormalExecution` ?

Reply:
it's kind of an "integration" test but HappyPath sounds good as well :wink: